larq_zoo/utils.py

```diff
@@ -8,8 +8,6 @@
 from keras_applications.imagenet_utils import _obtain_input_shape
 from collections import namedtuple
 
-ImagenetDataset = namedtuple("ImagenetDataset", ["input_shape", "num_classes"])
-
 
```

Comment:
Are you sure we do not need this anymore? If so the import should also be deleted.

Reply:
There were no references to this anymore in larq_zoo. As far as I could tell the tuple was used as a dummy dataset with only the relevant properties of `input_shape` and `num_classes`. Since `datasets` are no longer passed to models in the same way this should not be necessary anymore. d529090cf5d1272b7aa3438f1764d9660a948975 removes the import statement.